src/runtime/fetcher.js

```diff
@@ -40,6 +40,14 @@ export class Fetcher {
    * @param {!string|!Object} unusedData
    */
   sendBeacon(unusedUrl, unusedData) {}
+
+  /**
+   * POST data to a URL endpoint, get a Promise for a response
+   * @param {!string} unusedUrl
+   * @param {!string|!Object} unusedMessage
+   * @return {!Promise<!../utils/xhr.FetchResponse>}
+   */
+  postMessage(unusedUrl, unusedMessage) {}
```

Comment:
How about the name `sendPost`? I like `postMessage`, but it's already being used by an important JS API https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage

Reply:
Sure